docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM rocker/r-ver:4.0.5
```

Comment:
Yea, could do `4.1.0` for the time being because it is the latest that will have a stable MRAN snapshot.

Reply:
gonna leave it for now.. might change later.